bot/cogs/monstersurvey.py

```diff
@@ -0,0 +1,161 @@
+from discord.ext.commands import Bot, Context
+from asyncio import sleep as asleep
+from typing import Optional
+from discord.ext import commands
+from discord import Embed
+import logging
+import json
+import os
+
+log = logging.getLogger(__name__)
+
+EMOJIS = {
+    'SUCCESS': u'\u2705',
+    'ERROR': u'\u274C'
+}
+
+
+class MonsterSurvey:
+    """
+    Vote for your favorite monster!
+    This command allows users to vote for their favorite listed monster.
+    Users may change their vote, but only their current vote will be counted.
+    """
+
+    def __init__(self, bot: Bot):
+        """Initializes values for the bot to use within the voting commands."""
+        self.bot = bot
+        self.registry_location = os.path.join(os.getcwd(), 'resources', 'monstersurvey', 'monstersurvey.json')
+        with open(self.registry_location, 'r') as jason:
+            self.voter_registry = json.load(jason)
+
+    def json_write(self):
+        with open(self.registry_location, 'w') as jason:
+            json.dump(self.voter_registry, jason, indent=2)
+
+    @commands.group(
+        name='monster',
+        aliases=['ms']
+    )
+    async def monster_group(self, ctx: Context):
+        """
+        The base voting command. If nothing is called, then it will return an embed.
+        """
+
+        if ctx.invoked_subcommand is None:
+            default_embed = Embed(
+                title='Monster Voting',
+                color=0xFF6800,
+                description='Vote for your favorite monster!'
+            )
+            default_embed.add_field(
+                name='.monster show monster_name(optional)',
+                value='Show a specific monster. If none is listed, show a brief of all.',
```

Comment:
The alternative behavior described in this help statement doesn't happen. If no monster is provided, all the user is shown is the blank voting embed.